### ISSUE TITLE
Log indexer counters when  indexer terminates

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#1872]
A user fed quickwit with a json array of document instead of NDJSON.
The logs did not show any error.

Since the file was very small, quickwit cli also did not show any counter lines, with "Parse Errs", making it difficult for the user to identify the problem.

We should at the very least output the indexer counters when it finishes.

**Comment by fulmicoton** [comment]
```
2022-08-24T08:12:11.249Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=quickwit_indexing::actors::sequencer::Sequencer<quickwit_indexing::actors::publisher::Publisher>}: quickwit_actors::spawn_builder: actor-exit actor_id=quickwit_indexing::actors::sequencer::Sequencer<quickwit_indexing::actors::publisher::Publisher>-wispy-1giX exit_status=Success
2022-08-24T08:12:11.249Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=MergePublisher}: quickwit_actors::spawn_builder: actor-exit actor_id=MergePublisher-falling-e08s exit_status=Success
 Num docs       4 Parse errs     0 PublSplits   1 Input size     0MB Thrghput  0.00MB/s Time 00:00:02
 Num docs       4 Parse errs     0 PublSplits   1 Input size     0MB Thrghput  0.00MB/s Time 00:00:03
2022-08-24T08:12:14.201Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{msg_id=8}: quickwit_indexing::actors::indexing_pipeline: Indexing pipeline success. pipeline_id=IndexingPipelineId { index_id: "calendar", source_id: ".cli-ingest-source", node_id: "node-dark-dDbk", pipeline_ord: 0 } generation=1
2022-08-24T08:12:14.201Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}:: quickwit_actors::spawn_builder: actor-exit actor_id=IndexingPipeline-wild-KIrW exit_status=Success
 Num docs       4 Parse errs     0 PublSplits   1 Input size     0MB Thrghput  0.00MB/s Time 00:00:04
 Num docs       4 Parse errs     0 PublSplits   1 Input size     0MB Thrghput  0.00MB/s Time 00:00:04

Indexed 4 documents in 4s.
Now, you can query the index with the following command:
quickwit index search --index calendar --config ./config/quickwit.yaml --query "my query"
Clearing local cache directory...
2022-08-24T08:12:15.200Z  INFO quickwit_core::index: Clearing cache directory. path=./qwdata/indexing/calendar/.cli-ingest-source/cache
```

Here is the output that we have today. Offline @guilload mentionned that we already had a PR similar to that.
I am not sure if it forces the print of the summary line or if the PR never landed.

` Num docs       4 Parse errs     0 PublSplits   1 Input size     0MB Thrghput  0.00MB/s Time 00:00:04`

**Comment by boraarslan** [comment]
I think the mentioned PR is #1843. When I try to ingest the documents, I get an error message: 
```bash
-exit actor_id=MergeExecutor-fragrant-kffj exit_status=Success
2022-08-24T08:49:10.094Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=MergePackager}: quickwit_actors::spawn_builder: actor-exit actor_id=MergePackager-wandering-nS5i exit_status=Success
2022-08-24T08:49:10.094Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=MergeUploader}: quickwit_actors::spawn_builder: actor-exit actor_id=MergeUploader-red-d39f exit_status=Success
2022-08-24T08:49:10.094Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=quickwit_indexing::actors::sequencer::Sequencer<quickwit_indexing::actors::publisher::Publisher>}: quickwit_actors::spawn_builder: actor-exit actor_id=quickwit_indexing::actors::sequencer::Sequencer<quickwit_indexing::actors::publisher::Publisher>-weathered-BLyi exit_status=Success
2022-08-24T08:49:10.094Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{index=calendar gen=0}:{actor=MergePublisher}: quickwit_actors::spawn_builder: actor-exit actor_id=MergePublisher-proud-5m51 exit_status=Success
 Num docs       1 Parse errs     1 PublSplits   0 Input size     0MB Thrghput  0.00MB/s Time 00:00:02
 Num docs       1 Parse errs     1 PublSplits   0 Input size     0MB Thrghput  0.00MB/s Time 00:00:03
2022-08-24T08:49:12.932Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}::{msg_id=8}: quickwit_indexing::actors::indexing_pipeline: Indexing pipeline success. pipeline_id=IndexingPipelineId { index_id: "calendar", source_id: ".cli-ingest-source", node_id: "searcher-1", pipeline_ord: 0 } generation=1
2022-08-24T08:49:12.932Z  INFO {actor=quickwit_indexing::actors::indexing_service::IndexingService}:{msg_id=1}:: quickwit_actors::spawn_builder: actor-exit actor_id=IndexingPipeline-sparkling-p6Bq exit_status=Success
 Num docs       1 Parse errs     1 PublSplits   0 Input size     0MB Thrghput  0.00MB/s Time 00:00:04
 Num docs       1 Parse errs     1 PublSplits   0 Input size     0MB Thrghput  0.00MB/s Time 00:00:04

Indexed 0/1 documents in 4s (-99.0% success rate).
Clearing local cache directory...
2022-08-24T08:49:13.932Z  INFO quickwit_core::index: Clearing cache directory. path=./qwdata/indexing/calendar/.cli-ingest-source/cache
Command failed: Failed to ingest all the documents.
```

Mentioned PR is not included in v0.3.1. I think we can close the issue.
